src/m_misc.c

```diff
@@ -2262,6 +2264,20 @@ default_t defaults[] = {
   },
 #endif
 
+  {
```

Comment:
This needs `#ifdef _WIN32` guards.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/724/commits/6ddc5e75c6c51dcede3e755209a602d6e8b9ba96